ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -3189,6 +3194,12 @@ mod tests {
         let fmm = fmatcher("fp.128.x fp.128.y fp.128.x");
         assert!(fmm.matches("xmm0 xmm1 xmm0").is_ok());
         assert!(fmm.matches("xmm0 xmm0 xmm0").is_err());
+
+        let fmm = fmatcher("fp.128.x fp.128.y fp.128._");
```

Comment:
Does this mean we've seen f128 in the wild?

Reply:
The `xmm` registers are 128 bit, so we classify them as such, even if we only use them for doubles/floats.

Reply:
Ah, I see, it's just the register names. OK, cool.